candi/bashible/bundles/centos/node-group/031_install_containerd.sh.tpl

```diff
@@ -19,7 +19,7 @@ post-install() {
   systemctl daemon-reload
   systemctl enable containerd.service
 {{ if ne .runType "ImageBuilding" -}}
-  systemctl restart containerd.service
+  bb-flag-set containerd-need-restart
```

Comment:
Nitpicking: `needs`

Reply:
I don't think it matters because it's an internal flag

Reply:
i think this is not needed